### ISSUE TITLE
`kctrl`: Allow users to move all container images dependencies while releasing

### ISSUE THREAD

**Issue by 100mik** [carvel-dev/kapp-controller#952]
**Describe the problem/challenge you have**
On doing a package/package repo release, the Package and PackageRepo bundles are pushed to the user specified registry, however any pre-existing container images that the Package might be dependent on are not moved to the repository where the bundles reside.

**Describe the solution you'd like**
Users might or might not want to do this while releasing packages/package repos,  running an `imgpkg copy` for the bundle if the user specifies a flag (`--imgpkg-copy` or `--copy-dependencies` maybe). 

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

**Comment by 100mik** [comment]
Branched out from #632

**Comment by mamachanko** [comment]
I understand that "dependencies" are the images locked within `.imgpkg/images.ya?ml`?

**Comment by 100mik** [comment]
That is correct! `kctrl` would delegate that work to the `imgpkg copy` command
